runtime.go

```diff
@@ -63,6 +64,14 @@ type _runtime struct {
 
 func (self *_runtime) enterScope(scope *_scope) {
 	scope.outer = self.scope
+	if self.scope != nil {
+		scope.depth = self.scope.depth + 1
+	}
+
+	if self.stackLimit != 0 && scope.depth >= self.stackLimit {
+		panic(self.panicRangeError("RangeError: Maximum call stack size exceeded"))
```

Comment:
```
> (1).toString(40)
RangeError: RangeError: toString() radix must be between 2 and 36
    at toString (<native code>)
    at <anonymous>:1:2
```

Looks like you're correct - I'll fix this instance and put together another PR to fix the others.